app/javascript/components/lists/ListGroup.js

```diff
@@ -1,10 +1,13 @@
 import React, { useState, useEffect } from "react"
 import ListEntry from "./ListEntry"
+import PreviousDataLists from "../items/PreviousDatalists"
+import Table from "../items/Table"
 
 const ListGroup = (props) => {
   const [lists, setLists] = useState(props.lists);
 
   const csrf = props.csrf;
+  const previous_item_data = props.previous_item_data;
```

Comment:
Style nit:  you're using camelCase variables in the rest of the file, but snake_case here.  Since you're translating the data from props, it makes more sense to translate the variable name as well.

Reply:
Good catch...too many language syntax styles 😅